### PR TITLE
New ASCII Cheat Sheet

### PR DESCRIPTION
**Pull request by vikashvverma**
Displays a list of ASCII Code and corresponding character.

Screenshot:

screen shot 2016-02-19 at 11 14 01 pm

IA Page: [ https://duck.co/ia/view/ascii_code_cheat_sheet](https://duck.co/ia/view/ascii_code_cheat_sheet)

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/asciicode.json [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+        "sourceName": "Wikipedia",
+        "sourceUrl" : "https://en.wikipedia.org/wiki/ASCII"
+    },
+
+    "aliases": [
+        "ascii",
+        "ascii code"
+    ],
+
+    "template_type": "code",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/asciicode.json]: Hmm... `reference` would make more sense here.

**Comment by moollaza** [comment]
@vikashvverma Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@vikashvverma this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by mintsoft** [comment]
@moollaza I don't know why the tests didn't run on this PR, but they're broke! Fix incoming: https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2851

**Pull request merged**